scripts/ui/main_menu/main_menu.gd

```diff
@@ -0,0 +1,14 @@
+extends Control
+
+
+
+func _on_play_button_pressed():
+	get_tree().change_scene_to_file("res://scenes/ui/menu/play_menu.tscn")
+
+
+func _on_level_editor_button_pressed():
+	get_tree().change_scene_to_file("res://scenes/ui/menu/level_editor_menu.tscn")
```

Comment:
Is there a way to combine this menu scene with this scene: res://scenes/ui/level_editor/level_editor.tscn

Reply:
I was going to add another scene called selection scene where you can pick which level you want to edit. It will also have a button if you want to create a new level

Reply:
ok could u rename this scene to select_level_menu and the buttons too

Reply:
can you also add a screenshot or gif of the menu in the description

Reply:
![Screenshot 2024-07-26 185000](https://github.com/user-attachments/assets/0698e8cb-0371-44af-b5c9-5eb4e0cb3865)
![Screenshot 2024-07-26 185023](https://github.com/user-attachments/assets/f3a0405d-8239-4ea4-97df-2be9662c4e52)
![Screenshot 2024-07-26 185038](https://github.com/user-attachments/assets/dd6a4123-63b2-44af-b30f-815668acdcae)

I also renamed it